specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json

```diff
@@ -0,0 +1,548 @@
+    "operationId": "Key Phrases",
+    "consumes": [
+     "application/json"
+    ],
+    "produces": [
+     "application/json"
+    ],
+    "parameters": [
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     },
+     {
+      "name": "input",
+      "in": "body",
+      "description": "Collection of documents to analyze. Documents can now contain a language field to indicate the text language",
+      "required": true,
+      "schema": {
+       "$ref": "#/definitions/MultiLanguageBatchInputV2"
+      }
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful response results in 0 or more key phrases identified in each valid document",
+      "schema": {
+       "$ref": "#/definitions/KeyPhraseBatchResultV2"
+      },
+      "examples": {
+       "application/json": {
+        "documents": [
+         {
+          "keyPhrases": [
+           "world",
+           "input text"
+          ],
+          "id": "1"
+         },
+         {
+          "keyPhrases": [
+           "monde"
+          ],
+          "id": "2"
+         },
+         {
+          "keyPhrases": [
+           "carretera",
+           "tráfico",
+           "día"
+          ],
+          "id": "3"
+         }
+        ],
+        "errors": null
+       }
+      }
+     },
+     "default": {
+      "description": "Error Response",
+      "schema": {
+       "$ref": "#/definitions/ErrorResponse"
+      }
+     }
+    },
+    "deprecated": false
+   }
+  },
+  "/v2.0/languages": {
+   "post": {
+    "tags": [
+     "Detect Language"
+    ],
+    "summary": "The API returns the detected language and a numeric score between 0 and 1. \r\n            Scores close to 1 indicate 100% certainty that the identified language is true. \r\n            A total of 120 languages are supported.",
+    "operationId": "Detect Language",
+    "consumes": [
+     "application/json"
+    ],
+    "produces": [
+     "application/json"
+    ],
+    "parameters": [
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     },
+     {
+      "name": "input",
+      "in": "body",
+      "description": "Collection of documents to analyze.",
+      "required": true,
+      "schema": {
+       "$ref": "#/definitions/BatchInputV2"
+      }
+     },
+     {
+      "name": "numberOfLanguagesToDetect",
+      "in": "query",
+      "description": "(Optional. Deprecated) Number of languages to detect. Set to 1 by default. Irrespective of the value, the language with the highest score is returned.",
+      "required": false,
+      "type": "integer",
+      "format": "int32"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call results in the detected language with the highest probability for each valid document",
+      "schema": {
+       "$ref": "#/definitions/LanguageBatchResultV2"
+      },
+      "examples": {
+       "application/json": {
```

Comment:
Consider moving examples into separate folders and make the swagger smaller and cleaner. https://github.com/Azure/azure-rest-api-specs/blob/current/documentation/x-ms-examples.md